webportal/webapp/src/environments/environment.test.ts

```diff
@@ -0,0 +1,22 @@
+// The file contents for the current environment will overwrite these during build.
+// The build system defaults to the dev environment which uses `environment.ts`, but if you do
+// `ng build --env=prod` then `environment.prod.ts` will be used instead.
+// The list of which env maps to which file can be found in `.angular-cli.json`.
+
+export const environment = {
```

Comment:
This got pulled in from master (and I tweaked it to work with the new Nginx proxy naming conventions).